tools/extra/qpafilter/qpafilter.py

```diff
@@ -97,10 +97,23 @@ def __init__(self, args):
     def filter(self, items):
         for i in items:
             critical = float(i['critical'])
-            warning = (critical * self.args.warn_temp) / self.args.crit_temp
+            units = i['units']
+
+            if 'override' in i:
+                override = float(i['override'])
+                warning = (override * critical) / 100.0
+            else:
+                override = 0.0
+                warning = (critical * self.args.warn_temp) / self.args.crit_temp
+
             i['warning'] = str(warning)
 
-            print('{} w:{} c:{}'.format(i['label'], i['warning'], i['critical']))
```

Comment:
any reason why we don't want logger here?

Reply:
Nope.  The use of print is intentional.  It reminds me that we need to figure out what needs to be printed there.